src/panels/lovelace/hui-root.ts

```diff
@@ -120,18 +121,51 @@ class HUIRoot extends hassLocalizeLitMixin(LitElement) {
       .narrow="${this.narrow}"
     ></hui-notification-drawer>
     <ha-app-layout id="layout">
-      <app-header slot="header" effects="waterfall" fixed condenses>
+      <app-header slot="header" effects="waterfall" class="${classMap({
+        "edit-mode": this._editMode,
+      })}" fixed condenses>
         ${
           this._editMode
             ? html`
-                <app-toolbar>
+                <app-toolbar class="edit-mode">
                   <paper-icon-button
                     icon="hass:close"
                     @click="${this._editModeDisable}"
                   ></paper-icon-button>
                   <div main-title>
-                    ${this.localize("ui.panel.lovelace.editor.header")}
+                    ${
+                      this.config.title ||
+                        this.localize("ui.panel.lovelace.editor.header")
+                    }
+                    <paper-icon-button
+                      icon="hass:pencil"
+                      class="edit-icon"
+                      @click="${this._editLovelace}"
+                    ></paper-icon-button>
                   </div>
+                  <paper-menu-button
+                    no-animations
+                    horizontal-align="right"
+                    horizontal-offset="-5"
+                  >
+                    <paper-icon-button
+                      icon="hass:dots-vertical"
+                      slot="dropdown-trigger"
+                    ></paper-icon-button>
+                    <paper-listbox
+                      @iron-select="${this._deselect}"
+                      slot="dropdown-content"
+                    >
+                      <paper-item @click="${this._editModeDisable}"
+                        >Close UI editor</paper-item
+                      >
+                      <paper-item @click="${this.lovelace!.enableFullEditMode}"
+                        >Raw config editor</paper-item
+                      >
+                      <paper-item>Add custom resources</paper-item>
+                      <paper-item @click="${this._handleHelp}">Help</paper-item>
```

Comment:
We should move help to a paper-icon-button that lives next to the menu.

Reply:
Is that for both views or only edit?